Dockerfile

```
@@ -22,7 +22,7 @@ CMD ["poetry", "run", "flask", "run", "--host", "0.0.0.0", "-p", "5100"]
 # Production
 FROM base as production
 EXPOSE 5000
-CMD ["poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()", "--access-logfile", "gunicorn-access.log", "--log-file", "gunicorn.log"]
+CMD poetry run gunicorn -b 0.0.0.0:$PORT 'todo_app.app:create_app()' --access-logfile gunicorn-access.log --log-file gunicorn.log
```

You should cover the case where `$PORT` isn't present (i.e. when running it locally):
```suggestion
ENV PORT=5000
CMD poetry run gunicorn -b 0.0.0.0:$PORT 'todo_app.app:create_app()' --access-logfile gunicorn-access.log --log-file gunicorn.log
```

---

Done